content/optimization/importance.md

```diff
@@ -1,10 +1,284 @@
+
+Impurity gain gives us insight into the importance of a decision. In particular, larger $\Delta I$ indicates a more important decision. If some feature $(x_n)_d$ is the basis for several decision splits in a decision tree, the sum of impurity gains at these splits gives insight into the importance of this feature. Accordingly, one measure of the feature importance of $d$ as the average (with respect to the total number of internal nodes) impurity gain imparted by decision split on $d$. This method generalizes to the case of BDTs, in which case one would average this quantity across all weak learner trees in the ensemble. 
+
+Note that though decision trees are based on the feature $d$ producing the best (maximum impurity gain) split at a given branch node, *surrogate splits* are often used to retain additional splits corresponding to features other than $d$. Denote the feature maximizing the impurity gain $d_1$ and producing a split boundary $\delta_1$. Surrogte splitting involves tracking secondary splits with boundaries $\delta_2, \delta_3,...$ corresponding to $d_2,d_3,...$ that have the highest correlation with the maximum impurity gain split. The upshot is that in the event that input data is missing a value at field $d_1$, there are backup decision boundaries to use, mitigating the need to define multiple trees for similar data. Using this generalized notion of a decision tree, wherein each branch node contains a primary decision boundary maximizing impurity gain and several additional surrogate split boundaries, we can average the impurity gain produced at feature field $d$ over all its occurances as a decision split or a surrogate split. This definition of feature importance generalizes the previous to include additional correlations. 
+
+# Additional References
+## SHAP
+Recently, the Shapley Additive Explanations (SHAP, see [A Unified Approach to Interpreting Model Predictions](https://arxiv.org/abs/1705.07874)) framework was introduced as generalized feature importance framework for any prediction model. The framework is validated theoretically and empirically in its original paper and is currently [available via git](https://github.com/slundberg/shap#citations). 
+
+## Perturbation Ranking
```

Comment:
Do you plan to do anything with the following 2 sources? If not, perhaps you can comment these out? I don't want to lose the references, but I also don't want references without any context.

Reply:
Nope, removed them!